src/skmultiflow/transform/standard_scaler.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+
+from skmultiflow.transform.base_transform import StreamTransform
+from skmultiflow.utils import FastBuffer, get_dimensions
+
+
+class StandardScaler(StreamTransform):
```

Comment:
The method name should make explicit that this method is based on data windows. This is to avoid false expectations in the context of incremental learning.